apps/authorization/routers.py

```diff
@@ -44,7 +44,7 @@ async def login(
                           response_model=JSENDOutSchema[List[AvailableRolesOut]],
                           summary="Get available roles",
                           responses={200: {"description": "Successful get list of available roles response"}},
-                          tags=["SuperAdmin dashboard"])
+                          tags=["User dashboard"])
```

Comment:
why did u define "user dasboard", if the file is located in the authorization app?